vortex-filtering/include/vortex_filtering/probability/multi_var_gauss.hpp

```diff
@@ -0,0 +1,119 @@
+#pragma once
+#include <Eigen/Dense>
+#include <random>
+
+namespace vortex {
+namespace prob {
+
```

Comment:
Might as well use `namespace vortex::prob` for all of these (although I guess that is more of a styling choice..)

Reply:
missed this one from when I was writing this in c++14

Reply:
The future is now